modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js

```diff
@@ -71,72 +71,66 @@ const TranslationAdminContent = ({
 			</ClayModal.Header>
 
 			<ClayModal.Header withTitle={false}>
-				<ClayModal.Title>
-					<ClayInput.Group className="align-items-center">
-						<ClayInput.GroupItem>
-							<ClayInput
+				<ClayInput.Group className="align-items-center">
+					<ClayInput.GroupItem>
+						<ClayInput
+							aria-label={Liferay.Language.get('search')}
+							insetAfter={true}
+							onChange={(event) => {
+								const {value} = event.target;
+
+								setSearchValue(value);
+							}}
```

Comment:
If you want to decrease the quantity of lines here, you can:

```suggestion
							onChange={({target: value}) => 
								setSearchValue(value)
							}
```

Reply:
Done! Thanks ^^